apps/web-giddh/src/assets/css/style-1.scss

```diff
@@ -3229,3 +3229,6 @@ td.mat-footer-cell {
     transform: none !important;
 }
 /* material css for ledger end*/
+.float-left {
```

Comment:
@ravinderwalkover float-left is already a predefined class in bootstrap, we can remove this from here